readme.md

```diff
@@ -208,7 +208,7 @@ Then, go to [http://localhost:5000/](http://localhost:5000) and you should see a
 
 * **Question:** Try commenting out `body-parser` in your application. What happens and why?
 
-* **Your Answer:**
+* **Your Answer: I must have done somethign wrong, I did not see any changes
```

Comment:
If commented out you should loose access to req.body and therefore cannot send a proper response.